### PR TITLE
Tweak travis a bit for better patterns.

### PR DESCRIPTION
**Pull request by rnelson0**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,7 +11,9 @@ rvm:
   - 1.9.2
   - 1.9.3
   - 2.0.0
-  - 2.1.0
+  - 2.1.7
```
Comment (kylog) [inline .travis.yml]: I'd be tempted to go with 2.1.9. I don't expect any differences, but it's what puppet-agent ships with currently.

Comment (rnelson0) [inline .travis.yml]: Ah, funny, I cribbed 2.1.7 from the .travis.yml in puppetlabs/puppet :)

Comment (kylog) [inline .travis.yml]: Hahahahaha. Okay, PR coming there ...

**Comment by kylog** [comment]
LGTM!

**Comment by Taher-Ghaleb** [comment]
Hello,

Sorry, I know it's been a long time ago, but I am curious why caching hasn't been enabled before this (i.e., since December 17, 2014, when it became available to open source projects).

Thank you.

**Pull request merged**